<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ROI Data Tracker</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QMainWindow {
        background-color: #f5f7fa;
    }
    QLabel {
        color: #333;
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="sidebar">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
        QFrame#sidebar {
            background-color: #ffffff;
            border-right: 1px solid #e0e0e0;
        }
        QPushButton {
            text-align: left;
            padding: 12px 20px;
            border: none;
            background-color: transparent;
            font-size: 14px;
            color: #555;
            border-radius: 5px;
        }
        QPushButton:hover {
            background-color: #f0f2f5;
            color: #333;
        }
        QPushButton:checked {
            background-color: #e6f0ff;
            color: #0066cc;
            font-weight: bold;
        }
        QLabel#label_brand {
             padding: 20px;
             font-size: 18px;
             font-weight: bold;
             color: #222;
        }
       </string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_brand">
         <property name="text">
          <string>Tracker</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_nav_dashboard">
         <property name="text">
          <string>   Dashboard</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_nav_upload">
         <property name="text">
          <string>   Upload &amp; Review</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_nav_history">
         <property name="text">
          <string>   History</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>

       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>466</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_dashboard">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Dashboard (Placeholder - Switch to Upload)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_upload">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>40</number>
        </property>
        <property name="topMargin">
         <number>40</number>
        </property>
        <property name="rightMargin">
         <number>40</number>
        </property>
        <property name="bottomMargin">
         <number>40</number>
        </property>
        <item>
         <widget class="QLabel" name="label_header">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>24</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Upload and Review</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_desc">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #666;</string>
          </property>
          <property name="text">
           <string>Drag and drop your hygrometer photo here, or click to select a file.</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_dropzone">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>150</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
                QFrame#frame_dropzone {
                    border: 2px dashed #b0c4de;
                    border-radius: 10px;
                    background-color: #fafdff;
                }
           </string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_drop_icon">
             <property name="text">
              <string>ðŸ“·</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="font">
                <font>
                    <pointsize>32</pointsize>
                </font>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_drop_text">
             <property name="text">
              <string>Drag and drop your photo here</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_btn">
                <item>
                     <spacer name="horizontalSpacer_btn1"><property name="orientation"><enum>Qt::Horizontal</enum></property></spacer>
                </item>
                <item>
                    <widget class="QPushButton" name="btn_select_photo">
                     <property name="cursor">
                      <cursorShape>PointingHandCursor</cursorShape>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">
                        QPushButton {
                            background-color: #e0e4e8;
                            border: none;
                            padding: 8px 16px;
                            border-radius: 4px;
                            font-weight: bold;
                            color: #333;
                        }
                        QPushButton:hover {
                            background-color: #d0d4d8;
                        }
                      </string>
                     </property>
                     <property name="text">
                      <string>Select Photo</string>
                     </property>
                    </widget>
                </item>
                <item>
                     <spacer name="horizontalSpacer_btn2"><property name="orientation"><enum>Qt::Horizontal</enum></property></spacer>
                </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_review">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Review and Confirm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_review">
          <property name="styleSheet">
           <string notr="true">
            QFrame#frame_review {
                background-color: white;
                border-radius: 8px;
                border: 1px solid #e0e0e0;
            }
           </string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QScrollArea" name="image_scroll_area">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>300</height>
              </size>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <property name="styleSheet">
                <string notr="true">border: none;</string>
            </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="lbl_image_preview">
                 <property name="text">
                  <string>Preview will appear here</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="styleSheet">
                    <string notr="true">color: #aaa;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_results">
             <property name="horizontalSpacing">
              <number>20</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Temperature (Â°C)</string>
               </property>
               <property name="styleSheet">
                <string notr="true">font-weight: bold; color: #444;</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Humidity (%)</string>
               </property>
               <property name="styleSheet">
                <string notr="true">font-weight: bold; color: #444;</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLineEdit" name="input_temp">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">
                    QLineEdit {
                        border: 1px solid #d0d0d0;
                        border-radius: 4px;
                        padding: 5px;
                        font-size: 16px;
                    }
                </string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="input_humidity">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">
                    QLineEdit {
                        border: 1px solid #d0d0d0;
                        border-radius: 4px;
                        padding: 5px;
                        font-size: 16px;
                    }
                </string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="lbl_status">
              <property name="text"><string>Ready</string></property>
              <property name="styleSheet"><string>color: #888;</string></property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_confirm">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">
                QPushButton {
                    background-color: #007bff;
                    color: white;
                    border-radius: 4px;
                    font-weight: bold;
                    border: none;
                }
                QPushButton:hover {
                    background-color: #0069d9;
                }
             </string>
            </property>
            <property name="text">
             <string>Confirm &amp; Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
